test/server/reporter-test.js

```diff
@@ -98,7 +116,10 @@ describe('Reporter', function () {
             test:              testMocks[0],
             unstable:          true,
             browserConnection: browserConnectionMocks[0],
-            errs:              []
+            errs:              [],
+            quarantine:        {
+                attempts: [['1', '2'], []]
```

Comment:
Should I check this also? Will users face this? If so, what does it mean?

Reply:
no, this is only for testing

Reply:
okay then, let it be